app/components/paperShow/components/sourceButton.tsx

```diff
@@ -66,7 +66,7 @@ const SourceButton: React.FunctionComponent<SourceButtonProps> = props => {
               }}
             >
               <Icon icon="EXTERNAL_SOURCE" />
-              <span>View in Source</span>
+              <span>{paper.urls.length > 0 ? 'Sources' : 'Source'}</span>
```

Comment:
It should be like
`{paper.urls.length > 1 ? 'Sources' : 'Source'}` or
`Source{paper.urls.length > 1 && 's'}`

Reply:
👁

Reply:
Ok, that's my mistake

I fixed it